### ISSUE TITLE
Wish lists

### ISSUE THREAD

**Issue by veloek** [listribute/listribute#20]
One of the main ideas for this app is to work for wish lists. This kind of lists are rather special in that it shouldn't allow the creator to see item's checked status nor allow other users to add new items. Everyone except the creator should be able to see other's check marks to know what has already been taken off the list.

**Comment by eirikwah** [comment]
For this to work, I suggest (later on?) adding a public view of all the users that have read access to the list (so that the person that created the list can not simply create another dummy-user with read access to the checked status to "cheat").

**Comment by veloek** [comment]
Yes, we might have to consider the "security" aspect of it as well, but I don't think we should go to any extreme length to avoid "cheating". It will have to rely on some level of trust as well.

Like any other wish list you create in a physical store, you can always go and check what people have bought if you really want to know, but that's usually not a problem (unless you have very curious children perhaps :wink:).
